domains/info.vamtic.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "vamtic",
+    "email": "vamtic@yahoo.com"
+  },
+  "record": {
+    "CNAME": "vamtic.vercel.app"
```

Comment:
We don't support Vercel at the moment.

Reply:
![R](https://github.com/user-attachments/assets/210fe9e3-b8ca-4998-8e15-b8f4ba083070)